quilt/tools/command.py

```diff
@@ -876,3 +876,34 @@ def inspect(package):
     pkgobj = PackageStore.find_package(owner, pkg)
     if pkgobj is None:
         raise CommandException("Package {owner}/{pkg} not found.".format(owner=owner, pkg=pkg))
+
+    def _print_children(children, prefix, path):
```

Comment:
this code below is also causing unicode error when i run `quilt inspect`
the CRAZY thing is that on `master` branch, it looks like someone just gutted this whole functionality, see https://github.com/quiltdata/quilt-compiler/commit/f70d8993d53301b78a5c195467ad0979c9c72a16#diff-e97e2bbedcaad1ab3c091235193f000d

just glad that the version installed from `pip` still had a working version of this command

Reply:
Oh wow, I'm pretty sure that wasn't intentional. Good catch.